### PR TITLE
pylint: Fix inconsistent return statements

### PR DESCRIPTION
**Pull request by marusak**
Signed-off-by: Matej Marusak <mmarusak@redhat.com>

### CODE REVIEW

**Comment by mkutlak** [comment]
retest this please

File: src/webfaf/problems.py [hunk 1]
```diff
@@ -640,3 +640,4 @@ def bthash_forward(bthash=None):
                                problems=problems)
 
     abort(404)
+    return None
```
Comment (mkutlak) [inline src/webfaf/problems.py]: The `return None` will never be executed because the `abort(404)` will redirect you to an error page before it.

Comment (marusak) [reply src/webfaf/problems.py]: I know, but pylint doesn't. There are two options - this return or disable pylint warning on this function and I found this less harmful

Comment (mkutlak) [reply src/webfaf/problems.py]: You can change it to `return abort(404)`, that should stop pylint complaining.

**Comment by marusak** [comment]
@mkutlak updated

File: src/webfaf/reports.py [hunk 2]
```diff
@@ -347,6 +347,7 @@ def get_hash(os=None, release=None, since=None, to=None):
         return jsonify({"data": r_hash})
 
     abort(405)
+    return None
```
Comment (mkutlak) [inline src/webfaf/reports.py]: `return abort(405)`

**Comment by marusak** [comment]
@mkutlak updated

**Comment by mkutlak** [comment]
Thank you

**Pull request merged**